remove hover function

mirnawong1 opened the pull request:
this pr removes the hover function to the `detailsToggle` component and `FAQ` component. 

This pr will now only enable users to click the toggle to open/expand the question. 


![Screenshot 2024-01-30 at 20 22 30](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/05245dbc-1d36-4a32-a652-2761f60a3ad3)

![Screenshot 2024-01-30 at 20 22 21](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/b2dd7baf-8b4d-45bc-823b-ea8e0409abd6)

mirnawong1 committed:
Update index.js

update timeout so the the image hovers more consistently

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-22

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-22

mirnawong1 committed:
Merge branch 'remove-hover' into mirnawong1-patch-22

Commit:
remove hover

mirnawong1 committed:
Merge branch 'remove-hover' into mirnawong1-patch-22

Commit:
remove const

mirnawong1 committed:
Merge branch 'remove-hover' into mirnawong1-patch-22

mirnawong1 committed:
Merge branch 'current' into remove-hover

mirnawong1 committed:
Merge branch 'remove-hover' into mirnawong1-patch-22

nghi-ly reviewed (approved):
lgtm! 

non-blocking feedback: i really like the new look-and-feel. sugg removing the decorations that make it look like a link (color, underline, etc.). for example, the toggle widget in Notion

mirnawong1 committed:
Update index.js (#4781)

update timeout so the the image hovers more consistently

## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If related to an open 
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs,
see the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
<!--
Uncomment when publishing docs for a prerelease version of dbt:
- [ ] Add versioning components, as described in [Versioning
Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
- [ ] Review the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.
- [ ] For [docs
versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning),
review how to [version a whole
page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
and [version a block of
content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this
PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/static/_redirects`
- [ ] Run link testing locally with `npm run build` to update the links
that point to deleted pages

mirnawong1 committed:
Merge branch 'current' into remove-hover

JKarlavige reviewed (approved):
lgtm! :shipit:

mirnawong1 committed:
Merge branch 'current' into remove-hover
